charts/gitops-server/Chart.yaml

```diff
@@ -1,21 +1,6 @@
 apiVersion: v2
 name: weave-gitops
 description: Weave Gitops is a set of tools and services to aid your interactions with Flux
-# A chart can be either an 'application' or a 'library' chart.
-#
-# Application charts are a collection of templates that can be packaged into versioned archives
-# to be deployed.
-#
-# Library charts provide useful utilities or functions for the chart developer. They're included as
-# a dependency of application charts to inject those utilities and functions into the rendering
-# pipeline. Library charts do not define any templates and therefore cannot be deployed.
 type: application
-# This is the chart version. This version number should be incremented each time you make changes
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 4.0.36
-# This is the version number of the application being deployed. This version number should be
-# incremented each time you make changes to the application. Versions are not expected to
-# follow Semantic Versioning. They should reflect the version the application is using.
-# It is recommended to use it with quotes.
-appVersion: "v0.39.0-rc.2" # x-release-please-version
+version: 4.1.0-rc.1
+appVersion: v0.39.0-rc.2
```

Comment:
This change is not correct. Seems like release-please does not support monorepos including Helm. I'll try to find a workaround.